### ISSUE TITLE
Bookmark groups - A way to organize bookmarks within bookmarks tab

### ISSUE THREAD

**Issue by Ngoakor12** [Ngoakor12/coding-resource-finder#3]

**Comment by hrozienko** [comment]
Hello, can work on this issue?

**Comment by Ngoakor12** [comment]
> Hello, can work on this issue?

This is a big one, notice that I didn't mark it as a good first issue. I wanted to work on it myself. If I'm to let you work on it, just know that the review process might be a bit strict.

Still interested?

**Comment by ivoirh** [comment]
> > Hello, can work on this issue?
> 
> This is a big one, notice that I didn't mark it as a good first issue. I wanted to work on it myself. If I'm to let you work on it, just know that the review process might be a bit strict.
> 
> Still interested?

You're right, I was looking for something not too easy, but when I think about it more, it seems complicated, so you can work on it and I won't be interested in it anymore :)

**Comment by Ngoakor12** [comment]
> > > Hello, can work on this issue?
> > 
> > 
> > This is a big one, notice that I didn't mark it as a good first issue. I wanted to work on it myself. If I'm to let you work on it, just know that the review process might be a bit strict.
> > Still interested?
> 
> You're right, I was looking for something not too easy, but when I think about it more, it seems complicated, so you can work on it and I won't be interested in it anymore :)

No problem, I'll keep on adding issues so you might find something appropriate if you come back another time :)
